xpk.py

```diff
@@ -4559,8 +4564,9 @@ def validate_docker_image(docker_image, args) -> int:
   )
   if return_code != 0:
     xpk_print(
-        'Failed to validate your docker image, check the docker image. You'
-        f' should be able to navigate to the URL {docker_image} in {project}'
+        'Failed to validate your docker image, check that the docker image'
+        ' exists. If the docker image exists, the service account of this'
+        ' project maybe be missing the permissions to access the docker image.'
```

Comment:
nit to keep the variables: {docker_image} in {project} in the msg

Reply:
Addressed, thanks.